src/main/version.h

```diff
@@ -17,7 +17,7 @@
 
 #define FC_VERSION_MAJOR            3  // increment when a major release is made (big new feature, etc)
 #define FC_VERSION_MINOR            0  // increment when a minor release is made (small new feature, change etc)
-#define FC_VERSION_PATCH_LEVEL      1  // increment when a bug is fixed
+#define FC_VERSION_PATCH_LEVEL      2  // increment when a bug is fixed
```

Comment:
Yes, in general PRs should not change the version numbers - that should be done by whoever is preparing a release.